### PR TITLE
fix: Replaced FAB with ImageButton and removed elevation.

### LINKED ISSUE #495: remove elevation from share FAB in order completed fragment
working on it 👍 
screenshot_20180807-114201_eventyay attendee

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #495 

Changes: Replaced FAB with ImageButton and removed elevation. Also removed redundant line defining ScaleType in OrderCompletedFragment.kt which has already been defined in layout file fragment_order.xml.

Screenshots for the change:

### CODE REVIEW

File: app/src/main/res/layout/fragment_order.xml [hunk 1]
```diff
@@ -151,12 +151,11 @@
...
 
-                <com.google.android.material.floatingactionbutton.FloatingActionButton
+                <ImageButton
                     android:id="@+id/share"
                     android:layout_width="@dimen/fab_width"
                     android:layout_height="@dimen/fab_height"
                     android:layout_margin="@dimen/layout_margin_medium"
-                    android:background="@android:color/white"
                     android:scaleType="center"
                     app:backgroundTint="@android:color/white"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_order.xml]: Unneeded

File: app/src/main/res/layout/fragment_order.xml [hunk 2]
```diff
@@ -151,12 +151,11 @@
...
                     android:textColor="@color/black" />
 
-                <com.google.android.material.floatingactionbutton.FloatingActionButton
+                <ImageButton
                     android:id="@+id/share"
                     android:layout_width="@dimen/fab_width"
                     android:layout_height="@dimen/fab_height"
                     android:layout_margin="@dimen/layout_margin_medium"
-                    android:background="@android:color/white"
                     android:scaleType="center"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_order.xml]: Unneeded

**Comment by iamareebjamal** [comment]
There's no need to close a PR, you just need to push new commits to the branch

**Comment by angmas1** [comment]
> There's no need to close a PR, you just need to push new commits to the branch

I thought that was against the policy written in readme.md that says one commit per pull request, Sorry, won't close next time.

**Comment by iamareebjamal** [comment]
One PR per issue

File: app/src/main/res/layout/fragment_order.xml (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/res/layout/fragment_order.xml]: Change to `android:background="@android:color/transparent"`

**Comment by addiegupta** [comment]
You need to squash your commits. One PR should contain one commit only

**Pull request merged**